### ISSUE TITLE
The divine name

### ISSUE THREAD

**Issue by DavidHaslam** [lb42/KJV_1611#7]
Modern editions of the KJV use [small-caps](https://en.wikipedia.org/wiki/Small_caps) to style the **divine name** where it corresponds to the [tetragrammaton](https://en.wikipedia.org/wiki/Tetragrammaton) and a few cognate words.

- A case sensitive whole word search for `LORD` gave only **182** matches.
- A case sensitive whole word search for `Lord` gave **8276** matches.
- Total = 8458.

Compare similar searches for the SWORD **KJVA** module:
- `LORD` has **6574** matches
- `Lord` has **1875** matches
- Total = 8449.

This is based on the **diatheke** output which converts the internal OSIS markup to capitals.
cf. OSIS has a **divineName** element to identify such words, such that they can be suitable styled.

The totals differ only by 9, and are probably easily reconciled.

The issue here is that the HTML source text has not faithfully represented the printed black letter capitalisation of the divine name in a large proportion of cases, or so it seems.

NB. I've ignored _possessives_ in this comparison, but the analysis could be readily extended.

**Comment by lb42** [comment]
Yes, it does look as if the HTML normalises "LORD" ( blackletter caps usually with additional spacing)  to "Lord", most of the time. Not sure what to do about that. Are there any occurrences of "Lord" which don't correspond with the tetragrammaton? Is there anything special about the 182 cases where the HTML transcribers did choose to use caps? Nearly all of them are in Genesis.

**Comment by DavidHaslam** [comment]
Occurrences of 'Lord' in the NT are generally styled as Proper Case.
These are translations of the Greek word `κυριος` (if I keyed it correctly), not the Hebrew `YHWH`.

**Comment by DavidHaslam** [comment]
The 182 instances that are capitalised in the HTML are possibly because the transcription effort was divided into several volunteers. Unless it was all done by one individual who forgot, perhaps intending to apply a systematic post-processing change to all occurrences in the 39 books translated from the Hebrew OT.

Maybe he had realised by then that special style would be needed for small caps?
I.e. To match how it's rendered in much later editions that stopped using Blackletter.

I should supply separate comparison counts from the DC books to help us elucidate.
Some of these books are translations from the LXX.

**Comment by lb42** [comment]
Added a comment to the ODD doc to alert people to this and other transcription inconsistencies. Should we turn all the LORDs into Lords?

**Comment by DavidHaslam** [comment]
I think the answer depends on whether TEI has anything equivalent to the OSIS element **divineName**.

OSIS syntax is `<divineName>Lord</divineName>` which leaves the rendering engine to display the word with small-caps.

NB. For possessives, it should be `<divineName>Lord’s</divineName>` in later editions.
cf. KJV_1611 didn't have apostrophes for possessives.

Longer term, if the HTML on their website is never corrected in this matter, one route forward would be to do a direct word level comparison with the Blayney KJV of 1769. I have in mind something along this line. As for method, I have developed a novel use of Excel™ with a cunning formula. But that's for another day.

**Comment by lb42** [comment]
"Lord" in smallcaps does not look like "LORD" in small caps. I think you may be missing my point.

**Comment by DavidHaslam** [comment]
Yes - they are different. If we have in mind to provide a TEI markup for the word, then they should indeed all be `Lord` or `Lords` (possessive and plural). 

Looking at the counted words list, I see one entry evidencing a missing space.
```
00223	lord
08183	Lord
00164	LORD
00001	Lordcame
00005	lorde
00018	Lorde
00009	lordes
00002	Lordes
00001	lordly
00037	lords
00137	Lords
00002	lordship
00001	Lordship
```

It's in **Jonah 3:1**. Nice catch?

**Comment by DavidHaslam** [comment]
Aside: Does TEI have any highlighting markup for **spaced letters**?

`<hi type="spaced letters">JEREMIAH</hi>` to render as `J E R E M I A H`

**Comment by lb42** [comment]
Yes. You could use @rend or @style or @rendition. See http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.global.rendition.html and http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ST.html#STGAre

**Comment by DavidHaslam** [comment]
Cool!

In view of the observed uses of spaced letters in certain items (e.g. `CHAP.` ),
should a plan be formed to mark these accordingly?

`CHAP.` would not only be an abbreviation, but excluding the period, should also be styled as spaced.

**Comment by DavidHaslam** [comment]
I just spotted another related issue:
- A search for `God` gave **5531** hits.
- A search for `god` gave **48** hits.
- A search for `GOD` gave only **6** hits.

However, in many places where it has the phrase "Lord GOD" in the blackletter, the second word is normally in uppercase, yet 
- A search for `Lord God` gave **552** hits.
- A search for `Lord GOD` gave only **5** hits.

All these searches were _case-sensitive_ and _whole-words_.

The 5 verses where they transcribed it aright read:
```
<ab n="28">And now, O Lord GOD, (thou art that God, and thy words be true, and thou hast promised this goodnesse vnto thy seruant.)<note> Ioh.17. 17.</note></ab>
<ab n="2">And Abram said, Lord GOD, what wilt thou giue me, seeing I goe childlesse? and the steward of my house is this Eliezer of Damascus.</ab>
<ab n="8">And he said, Lord GOD, whereby shal I know that I shall inherit it?</ab>
<ab n="19">Thine owne wickednesse shall correct thee, and thy backslidings shall reproue thee: know therefore and see, that it is an euill thing and bitter that thou hast forsaken the Lord thy God, and that my feare is not in thee, saith the Lord GOD of Hostes.<note> Isai.3.9. os.5.5.</note></ab>
<ab n="22">For though thou wash thee with nitre, and take thee much sope, yet thine iniquitie is marked before me, saith the Lord GOD.<note> Iob. 9.30.</note></ab>
```

**Comment by DavidHaslam** [comment]
I alluded to cognate words, one of which is `Iah`. This is found in **Psalm 68:4** which reads:
```
<ab n="4">Sing vnto God, sing praises to his Name: extoll him that rideth vpon the heauens, by his Name Iah, and reioyce before him.</ab>
```
cf. In modern editions, 'JAH' is usually styled with capitals. Here's the 1611 blackletter:

![screenshot 2017-10-31 15 28 26](https://user-images.githubusercontent.com/16325414/32232633-60cbf02c-be50-11e7-8889-bd3f63ac5e9e.png)

@lb42  I am adding this to the list being prepared as per issue #15

**Comment by DavidHaslam** [comment]
FYI. These are the search results for `Iehouah`:
```
<ab n="35">It was round about eighteene thousand measures, and the name of the citie from that day shall be, The Lord is there.<note> Heb. Iehouah Shammah.</note></ab>
<head>1 Abraham is tempted to offer Isaac. 3 Hee giueth proofe of his faith and obedience. 11 The Angel stayeth him. 13 Isaac is exchanged with a ramme. 14 The place is called Iehouah-ijreh. 15 Abraham is blessed againe. 20 The generation of Nahor vnto Rebekah.</head>
<ab n="14">And Abraham called the name of that place Iehouah-ijreh, as it is said to this day, In the Mount of the LORD it shalbe seene.<note> That is, The LORD will see, or, prouide.</note></ab>
<ab n="4">Trust ye in the Lord for euer: for in the Lord Iehouah is euerlasting strength.<note> Heb. the rocke of ages.</note></ab>
<ab n="6">In his dayes Iudah shalbe saued, and Israel shall dwell safely, and this is his Name whereby hee shall be called, The Lord Our Righteousnes.<note> Deut. 33. 28. , Hebr. Iehouah-tsidkenu.</note></ab>
<head>1 The Israelites for their sinne are oppressed by Midian. 8 A prophet rebuketh them. 11 An Angel sendeth Gideon for their deliuerance. 17 Gideons Present is consumed with fire. 24 Gideon destroyeth Baals altar, and offreth a sacrifice vpon the altar Iehouah-shalom. 28 Ioash defendeth his son, &amp; calleth him Ierubbaal. 33 Gideons armie. 36 Gideons signes.</head>
<ab n="24">Then Gideon built an Altar there vnto the Lord, and called it Iehouah shalom: vnto this day it is yet in Ophrah, of the Abi-Ezrites.<note> That is, The Lord send peace.</note></ab>
```

My counted words list excluded chapter descriptions, so this unexpected spelling in **Genesis 22:14** wasn't noted previously:
Should `Iehouah-ijreh` [sic] have been `Iehouah-jireh` ?  This is down to the printers in 1611.
```
<ab n="14">And Abraham called the name of that place Iehouah-ijreh, as it is said to this day, In the Mount of the LORD it shalbe seene.<note> That is, The LORD will see, or, prouide.</note></ab>
```
![screenshot 2017-10-31 16 12 29](https://user-images.githubusercontent.com/16325414/32235097-726bc13a-be56-11e7-86d0-dd94b71ab16b.png)
